### DISCUSSION TITLE
How to add static assets with the WebService?

### DISCUSSION THREAD

**Discussion by danilomo** [eclipse-edc/Connector#993] [Q&A] [answered]
I want serve a small web application hosted inside the connector so administrators could edit assets and contracts. Tried to follow some tutorials about Jakarta (adding a /webapp/resources and WEB-INF folders), but did not succeed in this task.

**Comment by jimmarino** [comment, marked as answer]
The connector should not serve web applications or a UI. It does provide an API that can be accessed by a UI. 

Instead, I would create a single-page web app and deploy it using a cluster of HTTP servers or on a CDN and enable CORS in the connector.

**Reply by danilomo** [reply]
Fair enough! Thanks for the answer.
